src/App.js

```diff
@@ -1,25 +1,74 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import { Card, Row, Col, Divider, Input, Button } from 'antd';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Searchbar from './components/Searchbar';
 
 function App() {
+  const [food, setFood] = useState(foods);
+  const [alwaysFood, setAlwaysFood] = useState(foods);
+  const [visible, setVisible] = useState(true);
+  const [isFoodArrayEmpty, setIsFoodArrayEmpty] = useState(false);
+
+  const addToList = (item) => {
+    const foodCopy = [item, ...food];
+
+    setFood(foodCopy);
```

Comment:
Your create food is not working because here you're updating `food` with `setFood` but the variable that you're using to loop through and display all the foods is `alwaysFood` so you need to update the `alwaysFood` with `setAlwaysFood` as well:
```
  const addToList = (item) => {
    const foodCopy = [item, ...food];
    setFood(foodCopy);
    setAlwaysFood(foodCopy);
  };
```